frontend/src/components/Workspace/FlowChart/FlowChartNode/AlgorithmNode.tsx

```diff
@@ -109,7 +109,11 @@ const AlgorithmNodeImple = memo(function AlgorithmNodeImple({
         </Grid>
       </Grid>
       <ButtonGroup>
-        <Button size="small" onClick={onClickParamButton}>
+        <Button
+          size="small"
+          onClick={onClickParamButton}
+          disabled={status !== NODE_RESULT_STATUS.SUCCESS}
```

Comment:
disabled condition would be
`status === NODE_RESULT_STATUS.PENDING`

Reply:
@ReiHashimoto yes, I revised the condition to "status === NODE_RESULT_STATUS.PENDING", pls check it again